crates/cargo-gpu/src/main.rs

```diff
@@ -63,7 +65,37 @@ mod spirv_cli;
 mod spirv_source;
 mod toml;
 
-fn main() {
+/// Central function to write to the user.
+#[macro_export]
+macro_rules! user_output {
+    ($($args: tt)*) => {
+        #[allow(
+            clippy::allow_attributes,
+            reason = "`std::io::Write` is only sometimes called??"
+        )]
+        #[allow(
+            clippy::useless_attribute,
+            reason = "`std::io::Write` is only sometimes called??"
+        )]
+        #[allow(
+            unused_imports,
+            reason = "`std::io::Write` is only sometimes called??"
+        )]
+        use std::io::Write as _;
+
+        #[expect(
+            clippy::non_ascii_literal,
+            reason = "CRAB GOOD. CRAB IMPORTANT."
+        )]
+        {
+            print!("🦀 ");
```

Comment:
😜

Reply:
We can totally change this BTW. I just used Ferris to as an example of what we could do.